doc/content2.tex

```diff
@@ -1,723 +0,0 @@
-        the same result as for other models.
-
-        \begin{figure}[H]
-            \begin{center}
-                \scalebox{0.65}[0.5]{\includegraphics[angle=0]{img/src/results-double-mse.pdf}}
-                \caption{Test results for triple exponential smoothing. Average values of MSE for all tests samples.}
-                \label{img:results-triple-mse}
-            \end{center}
-        \end{figure}
-
-        %%%%%%%%
-        \subsection{Overall Results} \label{sec:overall}
-        This section contains aggregated results for simple, double and triple exponential smoothing models.
-
-        Chart \ref{img:results-overall-r} highlight that only in four tests Data Mining models in average produces
-        higher MSE than alternative R models. Test case 200\,--\,12(12) shows the worst result which is 28\%
-        higher than benchmarked R model. This was discussed in Section \ref{sec:results-triple}.
-
-        Average for all test cases is equal to 2\%. In other words Data Mining models in average produced mean
-        squared error higher than 2\% compared to R models. Therefore Data Mining model's prediction capabilities
-        can be considered as R alternatives.
-
-        \begin{figure}[H]
-            \begin{center}
-                \scalebox{0.65}[0.5]{\includegraphics[angle=0]{img/src/results-overall-r.pdf}}
-                \caption{Aggregated test results for all models. Comparison to reference R model.}
-                \label{img:results-overall-r}
-            \end{center}
-        \end{figure}
-
-        Aggregated results for Data Mining mean squared errors are depicted in Chart \ref{img:results-overall-mse}. The
-        results are similar to the results discussed for each model. It shows that longer training set does not
-        improve forecasting accuracy. Further research could identify ideal size of training set. These tests
-        proved that longer training size does not imply better forecasts.
-
-        \begin{figure}[H]
-            \begin{center}
-                \scalebox{0.65}[0.5]{\includegraphics[angle=0]{img/src/results-overall-mse.pdf}}
-                \caption{Aggregated test results for all models. Average values of MSE for all tests samples.}
-                \label{img:results-overall-mse}
-            \end{center}
-        \end{figure}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Conclusion} \label{chap:conclusion}
-The goal of the master's thesis is to develop independent module in Java language for providing alert prediction
-capabilities for open source project Hawkular.
-
-%The thesis first describes various approaches for time series forecasting, and evaluates which methods are suitable
-%for Hawkular requirements. Follows review of existing monitoring solutions and libraries for time series modelling.
-%It is shown that any of existing libraries are not suitable for Hawkular, therefore a decision was made to implement
```

Comment:
Janka ked je ten text taky sivy znamena to ze je to komentar (nie je to vo vyslednej praci)